packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -741,7 +736,6 @@ function ensureRootIsScheduled(root: FiberRoot, currentTime: number) {
     );
   }
 
-  root.callbackId = newCallbackId;
```

Comment:
We used to update the root `callbackId` with the next lane, why don't we need to do anything with the next lane in the new version?

Reply:
It was only used by this function. Since we don't need it for the logic above anymore, we don't have to store it.